### ISSUE TITLE
Issues about importing this project

### ISSUE THREAD

**Issue by DavidTheExplorer** [ErrorxCode/JxInsta#4]
After I got an official response from Instagram4J creator, I decided to migrate my project to use this library.
However, I have a few questions before I even write the first line of code:

1.  Why do I also need to include the `OkHttp` dependency - why isn't it included in the artifact?

2. Why do I have to download the jar myself? That's Maven and Gradle exist.
I highly recommend posting Jitpack dependencies.

**Comment by ErrorxCode** [comment]
**_I appreciate that you understood :)_**
1 : You need to add okhttp dependency since this time the jar is distributed locally as .jar file which don't include `okhttp` libs
2 : The reason the library is distributed manually is bcz, at that time, the jitpack was not working (see [this](https://github.com/jitpack/jitpack.io/issues/6161) and [this](https://github.com/jitpack/jitpack.io/issues/3733)) now it is working back again. 

_PS : The further release will be on jitpack._

**Comment by DavidTheExplorer** [comment]
> **_I appreciate that you understood :)_**

I hope it was clear that I had nothing against you :)

> 1 : You need to add okhttp dependency since this time the jar is distributed locally as .jar file which don't include `okhttp` libs 

What's the problem with creating a "fat" jar that includes the dependencies? This is usually how libraries are published. 
Is that due to the fat jar's size?

> 2 : The reason the library is distributed manually is bcz, at that time, the jitpack was not working (see [this](https://github.com/jitpack/jitpack.io/issues/6161) and [this](https://github.com/jitpack/jitpack.io/issues/3733)) now it is working back again.
> 
> _PS : The further release will be on jitpack.

Awesome! Can you please keep this on _open_ and notify this issue once it's on jitpack?

**Comment by ErrorxCode** [comment]
Yeah, the reason that I did not create a `fat jar` is mainly because of size. One more reason not to do so is that it is generally good practice to give the user your library without transitive dependencies so he/she has full control over it.

Also, mostly developers will be using this library to create a client or API, which will already most probably include the `okhttp` dependencies. So re-including it in the .jar will be waste of space.
